src/MPI.jl

```diff
@@ -4,21 +4,29 @@ module MPI
 
 using Compat
 
-const depfile = joinpath(dirname(@__FILE__), "..", "deps", "src", "compile-time.jl")
-isfile(depfile) || error("MPI not properly installed. Please run Pkg.build(\"MPI\")")
-include(depfile)
+if @windows? true : false
```

Comment:
I think this can be written slightly more elegantly. Look e.g. at the `HDF5` package; there are constructs `@windows_only begin` etc.